forge/ee/lib/billing/index.js

```diff
@@ -40,16 +40,11 @@ module.exports.init = async function (app) {
     return {
         createSubscriptionSession: async (team, user = null) => {
             const billingIds = await team.getTeamBillingIds()
-            let teamPrice = billingIds.price
+            const teamPrice = billingIds.price
 
             // Use existing Stripe customer
             const existingLocalSubscription = await team.getSubscription()
-            if (existingLocalSubscription && existingLocalSubscription.isTrial()) {
-                // Currently in trial mode. Check for trial billing ids
-                if (billingIds.trialPrice) {
-                    teamPrice = billingIds.trialPrice
-                }
-            }
```

Comment:
When initiating the stripe session, we will no longer apply the 'trial price'. This means stripe will show the true cost of the subscription - and we don't stay in trial mode after creating the subscription.